client/src/components/ChallengeCard.tsx

```diff
@@ -2,15 +2,20 @@ import Card from "react-bootstrap/Card";
 import Button from "./Button.tsx";
 import { StarFill } from "react-bootstrap-icons";
 import { ChallengeDetailsShort } from "../types/ChallengeDetailsShort.ts";
-import { useMemo } from "react";
+import { useMemo, useState } from "react";
+import { ButtonToolbar } from "react-bootstrap";
 
 function ChallengeCard({
   title,
   generalDescription,
   id,
   difficulty,
-  completed
+  completed,
+  admin,
```

Comment:
is `admin` a boolean? I think `isAdmin` would be a better variable name if it is